README.md

```diff
@@ -1,70 +1,80 @@
-# PHP Study Guide
+# Guia de Estudos PHP
 
-## What?
+## O que é?
 
-Originally PHP Study Guide was a small project that provides step by step guide
-on information needed to pass [PHP5.3 ZCE certification][1].
+Originalmente o Guia de Estudos PHP era um projeto pequeno com um guia passo-a-passo
+com informações necessárias para passar na [certificação PHP5.3 ZCE][1].
 
-For now it aims to go little bit beyond the Zend Certification and is a good
-place to start your experience with PHP5.3 or, if you are already an experienced
-web developer, to return to the basics and remember some details.
+Agora o foco é ir um pouco além da certificação Zend e é um bom lugar para
+começar sua experiência com PHP5.3 ou, se você já é um desenvolvedor web experiente,
+para retornar ao básico e relembrar alguns detalhes.
```

Comment:
retomar os fundamentos e lembrar de alguns detalhes. -> (http://www.linguee.com.br/ingles-portugues/traducao/back+to+basics.html)